src/common/stats/StatsManager.cpp

```diff
@@ -86,7 +87,7 @@ int32_t StatsManager::registerHisto(folly::StringPiece counterName,
                 bucketSize,
                 min,
                 max,
-                StatsType(60, {seconds(60), seconds(600), seconds(3600)}))));
```

Comment:
Why change `numBuckets` to `720`, `numBuckets` is each level

Reply:
i thought `numBuckets` means time gap, `60` means statsmanager scrape metrics every minute. If we want to scape metrics every 5s, It needs 60 * 12 buckets.